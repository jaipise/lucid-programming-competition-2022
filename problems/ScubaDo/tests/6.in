iADeDhDGQk, tittueyiUX, LsJhSRNhEN, VpQiXdcHaK, tTHmvhcpUA, tAOxRWEzyf, qAHYyWNQbI, UpzOOsmrYj, ArTbnCvPGt, ygIHYZnvFl
9, 8, 3, 8, 9, 5, 8, 3, 9, 9
15
tcWVBvnPRUDZrfKAXVOH
LsJhSRNhEN, ygIHYZnvFl, VpQiXdcHaK, ArTbnCvPGt, UpzOOsmrYj, qAHYyWNQbI
7, 5, 7, 10, 8, 9
GNSEnEjVZCnKRrEuBLfd
tTHmvhcpUA, tittueyiUX, tAOxRWEzyf, UpzOOsmrYj, iADeDhDGQk
4, 6, 3, 3, 9
fxhuGRgwugoQsJyKTBpU
ArTbnCvPGt, UpzOOsmrYj, tAOxRWEzyf, qAHYyWNQbI, LsJhSRNhEN, tTHmvhcpUA, iADeDhDGQk, tittueyiUX, VpQiXdcHaK
4, 9, 1, 7, 9, 8, 2, 8, 4
LwUhIbwiLXNZvHleumCL
UpzOOsmrYj, tTHmvhcpUA, tittueyiUX, qAHYyWNQbI, VpQiXdcHaK
8, 2, 10, 8, 9
uBtOrBEWocvczKUbdhfe
ArTbnCvPGt, VpQiXdcHaK, UpzOOsmrYj, qAHYyWNQbI, tTHmvhcpUA, tAOxRWEzyf, iADeDhDGQk, ygIHYZnvFl, LsJhSRNhEN
5, 9, 8, 7, 4, 1, 1, 10, 6
dpfEIiEkPcASwJuhtUBJ
iADeDhDGQk, qAHYyWNQbI
1, 6
ygHSXvRppqyXsPWEUflR
UpzOOsmrYj
2
FdpwsaicpwIzqSxHwXtQ
iADeDhDGQk, ygIHYZnvFl, UpzOOsmrYj, tittueyiUX, LsJhSRNhEN, tTHmvhcpUA, VpQiXdcHaK, tAOxRWEzyf
4, 2, 1, 5, 7, 2, 4, 5
GAnaVofvCbwAwiJMUqev
tAOxRWEzyf, iADeDhDGQk, VpQiXdcHaK, qAHYyWNQbI, ArTbnCvPGt, LsJhSRNhEN, ygIHYZnvFl, UpzOOsmrYj, tittueyiUX
5, 6, 1, 9, 5, 6, 2, 6, 4
KIAOXSBLbwElmDudUuXg
LsJhSRNhEN, ygIHYZnvFl, qAHYyWNQbI, UpzOOsmrYj, tAOxRWEzyf
9, 4, 7, 9, 3
eKYVQfUQbbCGeUMoFjAu
tAOxRWEzyf, VpQiXdcHaK
2, 4
qlMqEoBVWArigvBAfKnc
qAHYyWNQbI, iADeDhDGQk, ygIHYZnvFl, UpzOOsmrYj, tittueyiUX, LsJhSRNhEN, ArTbnCvPGt, VpQiXdcHaK
6, 10, 1, 4, 2, 9, 6, 4
KNFWoiAxZKFXXhwsbDAs
tTHmvhcpUA, tittueyiUX, qAHYyWNQbI, ArTbnCvPGt, ygIHYZnvFl, VpQiXdcHaK, LsJhSRNhEN, iADeDhDGQk
8, 1, 10, 2, 1, 4, 9, 6
FnGeuwJxOjNAlNfiquao
ygIHYZnvFl
4
CBMgCJUeLOcRONdXIwLO
iADeDhDGQk, tittueyiUX, LsJhSRNhEN, VpQiXdcHaK, tTHmvhcpUA, tAOxRWEzyf, qAHYyWNQbI, UpzOOsmrYj, ArTbnCvPGt, ygIHYZnvFl
9, 8, 3, 8, 9, 5, 8, 3, 9, 9