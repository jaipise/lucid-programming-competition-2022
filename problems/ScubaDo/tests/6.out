CBMgCJUeLOcRONdXIwLO
dpfEIiEkPcASwJuhtUBJ
eKYVQfUQbbCGeUMoFjAu
FnGeuwJxOjNAlNfiquao
GNSEnEjVZCnKRrEuBLfd
ygHSXvRppqyXsPWEUflR
